includes/Classifai/Features/TitleGeneration.php

```diff
@@ -256,13 +256,6 @@ public function enqueue_admin_assets( string $hook_suffix ) {
 					);
 				}
 			}
-
-			wp_enqueue_style(
-				'classifai-language-processing-style',
-				CLASSIFAI_PLUGIN_URL . 'dist/language-processing.css',
-				[],
-				get_asset_info( 'language-processing', 'version' ),
-			);
```

Comment:
It seems the language processing CSS file contains some CSS of the time generation model. Removing this minor break style of the title generation model. Need to move those CSS to `classifai-plugin-classic-title-generation.css`.

![image](https://github.com/user-attachments/assets/22b1148f-def6-4c0c-973a-ca411f84852e)